packages/api-client/src/asyncOperations.ts

```diff
@@ -19,14 +19,15 @@ export const getAsyncOperation = async (params: {
 }): Promise<ApiGatewayLambdaHttpProxyResponse> => {
   const { prefix, asyncOperationId, callback = invokeApi } = params;
 
-  return await callback({
+  const response = await callback({
     prefix,
     payload: {
       httpMethod: 'GET',
       resource: '/{proxy+}',
       path: `/asyncOperations/${asyncOperationId}`,
     },
   });
+  return JSON.parse(response.body);
```

Comment:
Consider updating the function's description in the comment above for the return value

Reply:
eef1d9785f65c5981222a1846193cfa4a40df51e